### PR TITLE
Intentional style error.

### PR DESCRIPTION
**Pull request by ekpyron**
Just for checking the stackenbotten token.

### CODE REVIEW

**Comment by stackenbotten** [comment]
There was an error when running `chk_coding_style` for commit `3c459a51bf692fd65cc9a5f2f425239cde7ddb89`:
```
Error: Trailing whitespace found:
libsolidity/analysis/PostTypeChecker.cpp:80:        

```
Please check that your changes are working as intended.

File: libsolidity/analysis/PostTypeChecker.cpp [hunk 1]
```diff
@@ -77,6 +77,7 @@ void PostTypeChecker::endVisit(ErrorDefinition const& _error)
 
 bool PostTypeChecker::visit(EmitStatement const& _emit)
 {
+
```
Comment (stackenbotten) [inline libsolidity/analysis/PostTypeChecker.cpp:80]: Coding style error

**Comment by stackenbotten** [comment]
There was an error when running `chk_coding_style` for commit `dbf04e90794b956dcd40698b0d38f63e37c1d1e7`:
```
Error: Trailing whitespace found:
libsolidity/analysis/PostTypeChecker.cpp:80:        

```
Please check that your changes are working as intended.

File: libsolidity/analysis/PostTypeChecker.cpp (hunk 1 above)
Comment (stackenbotten) [inline libsolidity/analysis/PostTypeChecker.cpp:80]: Coding style error

**Comment by stackenbotten** [comment]
There was an error when running `chk_coding_style` for commit `8dcfd59410a083949c091d2e19cdd0ab4674d3de`:
```
Error: Trailing whitespace found:
libsolidity/analysis/PostTypeChecker.cpp:80:        

```
Please check that your changes are working as intended.

File: libsolidity/analysis/PostTypeChecker.cpp (hunk 1 above)
Comment (stackenbotten) [inline libsolidity/analysis/PostTypeChecker.cpp:80]: Coding style error

**Comment by ekpyron** [comment]
Seems to work with the new token.

**Pull request closed without merging**